internal/config/config.go

```diff
@@ -53,6 +53,7 @@ type Peer struct {
 	Confederation       *int      `yaml:"confederation" description:"BGP confederation (RFC 5065)" default:"-"`
 	ConfederationMember *bool     `yaml:"confederation-member" description:"Should this peer be a member of the local confederation?" default:"false"`
 	TTLSecurity         *bool     `yaml:"ttl-security" description:"RFC 5082 Generalized TTL Security Mechanism" default:"false"`
+	DefaultBGPLocalPref *int      `yaml:"default-bgp-local-pref" description:"Default value for local preference" default:"-"`
```

Comment:
For consistency with the existing `local-pref` attribute, lets rename this to `DefaultLocalPref` and `default-local-pref`.